test/test_readers.py

```diff
@@ -40,7 +40,7 @@ def test_read_n_chars(self):
 
         self.assertEquals(self.reader.read(1), '')
 
-    def test_read_n_chars(self):
+    def test_read_n_chars_2(self):
```

Comment:
there were 2 tests with the same name, separating them as `test_*_1` & `test_*_2`